CHANGELOG.md

```diff
@@ -1,4 +1,58 @@
 # Change Log
+
+## [0.29.0](https://github.com/confluentinc/ksql/releases/tag/v0.29.0-ksqldb) (2023-04-24)

```

Comment:
This date needs to be updated (to the release date, once we're ready to merge), and the new tag should be verified.